src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexReadOperation.cs

```diff
@@ -591,7 +591,7 @@ private IDisposable GetSort(IndexQueryServerSide query, Index index, Func<string
 
             if (orderByFields == null)
             {
-                if (query.Metadata.HasBoost == false && index.HasBoostedFields == false)
+                if (true || query.Metadata.HasBoost == false && index.HasBoostedFields == false)
```

Comment:
?

Reply:
Temporarily turned off. We sorted every time when boosting is involved. This has a great impact on query `from index 'indexName' `